lib/matplotlib/__init__.py

```diff
@@ -1294,6 +1294,13 @@ def is_interactive():
     return rcParams['interactive']
 
 
+def _val_or_rc(val, rc_name):
+    """
+    If *val* is None, return ``mpl.rcParams[rc_name]``, otherwise return val.
```

Comment:
```suggestion
    Return the value if present, otherwise lookup `rc_name` in the rcParams dictionary.
```
I'd suggest rewording this to be more human friendly since the code already explains what you had written.

Reply:
Using None and ``mpl.rcParams[rc_name]`` is good, because that's precise and comprehensible. Optional, slightly less algorithmic variant.

```suggestion
    Pass *val* through unless it is None, in which case return ``mpl.rcParams[rc_name]``.
```